tests/unittest.py

```diff
@@ -32,8 +32,14 @@ def shorten_hash(container):
     return container_type(h[-2:].hex() for h in container)
 
 
-def _load_peer_id_pool(file_path: str = 'tests/peer_id_pool.json') -> Iterator[PeerId]:
+def _load_peer_id_pool(file_path: Optional[str] = None) -> Iterator[PeerId]:
     import json
+
+    if file_path is None:
+        this_file_path = os.path.dirname(__file__)
```

Comment:
Maybe move it to a `get_default_peer_id_filepath(...)`?

Reply:
Done!

Reply:
Thanks!